### ISSUE TITLE
kid empty, unable to lookup correct key

### ISSUE THREAD

**Issue by Sutolix** [usefulteam/jwt-auth#74]
When I try to make a request using the authentication token I get the following error:

```js
{
	"success": false,
	"statusCode": 403,
	"code": "jwt_auth_invalid_token",
	"message": "\"kid\" empty, unable to lookup correct key",
	"data": []
}
```

**Comment by dominic-ks** [comment]
Hey @Sutolix, I've seen this error on sites where another WP plugin is using firebase/php-jwt 6.X and this plugin has been using 5.2 for a while, so it's likely clashing with that.

There should hopefully be an update to the php-jwt lib on this plugin soon which will make that go away.

**Comment by sun** [comment]
Seems like this has been with resolved with #76
